api/components/image.schema.json

```diff
@@ -12,16 +12,71 @@
             ]
         },
         "path": {
-            "description": "The image path from the resources folder",
+            "description": "The image path from the resources folder or from the network depending on the value of fromNetwork.",
             "type": "string"
         },
+        "fromNetwork": {
+            "description": "Whether the image will be loaded from a network url of from an application's resource path.",
```

Comment:
Cannot this be done by reading the path ? If it starts by http(s):// then it is from Network. If it is a classic path '/resources/images/myImage.jpg' then it is NOT from network ? 

This is a lot simpler for the dev IMO.

Reply:
I don't know if it is as simple as that. Can't a URL be valid without `http(s)://` ? For Example is this URL valid : `google.com` ? If it is, what is the difference between the local file `google.com` and the website `google.com` ?

I did not find any real information for this ...

Reply:
@Nesqwik  @shiipou

Reply:
I'm pretty sure it is possible to create a regex that can match a web URL. I don't think its a real issue to force the `http(s)://` before.

I still prefer the "fromNetwork" property over the two different components.

You can ask @taorepoara :)

Reply:
I also think that the fromNetwork is not needed. `google.com` is not an URL by definition but a domain and it's not enough.

We can change the `path` property to `src` ou `source` to be clearer, but one property could handle both situations.